index.html

```diff
@@ -149,10 +149,15 @@
 
 // Add cube mesh to your three.js scene
 scene.add(cube);
+
 // Request animation frame loop function
+var lastRender = 0;
 function animate(timestamp) {
+  var delta = Math.min(timestamp - lastRender, 500);
```

Comment:
Not a visible jump. The only thing is that the cube will not be axis
aligned on the first frame. If that's a concern you can just initialize
`lastRender = Date.now()`
On Jan 21, 2016 9:29 PM, "Boris Smus" <notifications@github.com> wrote:

> In index.html
> <https://github.com/borismus/webvr-boilerplate/pull/93#discussion_r50495881>
> :
>
> >  function animate(timestamp) {
> > +  var delta = Math.min(timestamp - lastRender, 500);
>
> Looks like this will lead to an immediate jump on load?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/borismus/webvr-boilerplate/pull/93/files#r50495881>.
>